tools/build-tools/src/typeValidator/testGeneration.ts

```diff
@@ -87,14 +87,25 @@ interface TestCaseTypeData extends TypeData{
 function buildTestCase(getAsType:TestCaseTypeData, useType:TestCaseTypeData, isCompatible: boolean){
     const getSig =`get_${getAsType.prefix}_${getFullTypeName(getAsType).replace(".","_")}`;
     const useSig =`use_${useType.prefix}_${getFullTypeName(useType).replace(".","_")}`;
+    const getIsRemoved = getAsType.kind.startsWith("Removed");
```

Comment:
can we avoid the string parse here and just add an `removed:boolean` field to TestCaseTypeData that will be true when it is removed

Reply:
The only issue is that we need to know if `currentType` is passed in as `getAsType` or `useType`. So, we may have to pass in two fields to this function, but that is probably still better than the string parse.